qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -216,6 +220,19 @@ class AerSimulator(AerBackend):
       values (16 Bytes). If set to 0, the maximum will be automatically
       set to the system memory size (Default: 0).
 
+    * ``cuStateVec_enable`` (bool): This option enables accelerating by
+      cuStateVec library of cuQuantum from NVIDIA, that has highly optimized
+      kernels for GPUs. This option is enabled when the number of qubits of
+      the input circuit is equal or greater than ``cuStateVec_threshold``.
+      Currently this option only works well for large number of qubits.
+      Also this option will be disabled for noise simulation
+      (Default: True).
```

Comment:
can we add "this option will be ignored if AerSimulator is not built with cuStateVec support" ?

Reply:
added